CMakeLists.txt

```diff
@@ -32,3 +35,9 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
 add_subdirectory(visual_marker_mapping)
 add_subdirectory(fast_mit_apriltags EXCLUDE_FROM_ALL)
+
+# add the library for umich AprilTags
+include_directories(${PROJECT_SOURCE_DIR}/umich_apriltags/)
```

Comment:
Mach bitte eine CMakeLists.txt in das umich_apriltags Projekt. Das passt da besser hin.